### PR TITLE
Controllers

### PR DESCRIPTION
**Pull request by yosefHesham**
### Adding Controllers and Views
---

- Created routes for user and posts
- Created Controllers to handle requests
- Created plain views

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -3,4 +3,10 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
```
Comment (t-yanick) [inline config/routes.rb]: - [ ] Great job working on this milestone. It is not professional to keep lines of commented codes in our codebase. kindly delete this commented line of code.

File: app/views/users/index.html.erb [hunk 2]
```diff
@@ -0,0 +1,12 @@
+<html>
```
Comment (t-yanick) [inline app/views/users/index.html.erb:1]: - [ ] Great job adding the Users index view page. As per project requirements, you are supposed to have a Users show view page just as you did for posts. kindly add this `# app/views/users/show.html.erb`

**Review by t-yanick — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED

Hi @yosefHesham , 

### Good job so far!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Controllers setup without using `rails generate scaffold`
:heavy_check_mark: Added actions to the controllers as per project requirements

There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️

- [ ] When I click on your license link, I am redireceted to an error 404 page. It is not professional to allow broken links in our codebase.
Screen Shot 2023-03-02 at 12 21 26 PM


- [x] Kindly edit your pull request title and make it more descriptive. Writing just `Controllers` is too vague.

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by victorokoroji — APPROVED** [review]
Hi @yosefHesham, 

# ✔️   Status: APPROVED 🏆 🎉 🎊 

You have done a great job on this project, keep it up 👍 🥇. Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
congratulations!  🎉💯🌟 

## Top highlight 🎯

- The Pull Request is following the Git Flow ✅ 
- The commit messages are meaningful ✅ 



## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**